static/css/custom.css

```diff
@@ -58,281 +86,181 @@ h2 {
-  font-size: 21px;
-  letter-spacing: 0.03em;
+.home-hero h1 {
+  font-weight: var(--ifm-font-weight-normal);
 }
 
-.inner {
-  max-width: 560px;
-  margin: auto;
+.home-hero p {
+  font-weight: var(--ifm-font-weight-normal);
+  margin: 0;
 }
 
-.splashFooter {
-  background: #000000;
-  color: #ffffff;
-  text-align: center;
-  padding: 45px 20px;
-  display: flex;
-  justify-content: center;
-  position: relative;
+.home-hero__tagline {
+  color: var(--ifm-color-gray-500);
+  line-height: 1.39;
 }
 
-.splashFooter > div {
-  max-width: 532px;
-  font-size: 17px;
+.home-hero__icon {
+  width: 120px;
+  margin-bottom: var(--ifm-paragraph-margin-bottom);
 }
 
-.splashFooter a {
-  color: #ffffff;
-  text-decoration: underline;
+.home-hero__footer {
+  background-color: var(--ifm-color-black);
+  padding: var(--space-m);
 }
 
-.contentBlock {
-  border-bottom: 1px solid #e3e5e9;
-}
-
-.contentContainer {
-  max-width: 532px;
-  margin: auto;
-  padding: 0 2em 5em 2em;
+.home-hero__footer a {
+  color: var(--ifm-color-white);
+  text-decoration: underline;
 }
 
-.contentContainer p {
-  color: #636975;
-  font-size: 17px;
+.home-section {
+  border-bottom: var(--ifm-hr-height) solid var(--ifm-hr-background-color);
 }
 
-.cardBlock {
+.home-section-header {
   text-align: center;
-  padding: 0 2em 5em 2em;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
-
-.cardBlock p {
-  color: #636975;
 }
 
-.cardContainer {
+.home-cards {
   display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
   flex-wrap: wrap;
-}
-
-.card {
-  font-size: 14px;
-  color: #636975;
-  display: flex;
+  gap: var(--space-s);
+  margin-bottom: var(--space-s);
   justify-content: center;
-  align-items: center;
-  padding: 14px;
-  width: 258px;
-  height: 110px;
-  box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.25);
-  border-radius: 4px;
-  margin: 8px;
-  transition: 200ms;
-}
-
-.card:hover {
-  transform: scale(1.05);
-}
-
-.card a {
-  color: dodgerblue;
 }
 
-.card img {
-  max-width: 100%;
-  max-height: 100%;
-  object-fit: contain;
-}
-
-.card-merge {
-  width: auto;
-  height: auto;
-  padding: 24px 24px 20px 24px;
-  flex-direction: row;
-}
-
-.card-merge-icon {
-  margin-right: 12px;
-}
-
-.card-wide {
-  width: 532px;
-  height: 130px;
+[class*='home-card--'] {
   display: flex;
-  flex-direction: row;
-  align-items: flex-start;
-  padding: 33px 24px;
-}
-
-.card-wide:hover {
-  text-decoration: none;
-}
-
-.card-wide-content {
-  text-align: left;
-  margin: 0 16px;
-  flex: 1;
+  align-items: center;
+  transition: box-shadow 0.6s cubic-bezier(0.165, 0.84, 0.44, 1);
+  position: relative;
+  overflow: initial;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.15);
+  border-radius: var(--ifm-global-radius);
 }
 
-.card-wide-content h4 {
-  margin: 0 0 8px 0;
-  font-size: 17px;
-  color: #000;
+[class*='home-card--'] {
+  padding: var(--space-s);
 }
 
-.card-wide-arrow {
+[class*='home-card--']::after {
```

Comment:
This is a performance-related choice, see: https://tobiasahlin.com/demo/animate-box-shadow/